src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -0,0 +1,292 @@
+import OlFeature from 'ol/feature';
+import OlGeomMultiPolygon from 'ol/geom/multipolygon';
+import OlGeomMultiPoint from 'ol/geom/multipoint';
+import OlGeomMultiLineString from 'ol/geom/multilinestring';
+import OlFormatGeoJSON from 'ol/format/geojson';
+
+import {
+  booleanPointInPolygon,
+  buffer,
+  centroid,
+  difference,
+  getCoords,
+  intersect,
+  multiLineString,
+  polygonize,
+  polygonToLine,
+  segmentEach,
+  union
+} from '@turf/turf';
+
+/**
+ * Helper Class for the geospatial analysis.
+ *
+ * @class GeometryUtil
+ */
+class GeometryUtil {
+
+  /**
+   * Splits a ol.feature with/or ol.geom.Polygon by a ol.feature with/or ol.geom.LineString
+   * into an array of instances of ol.feature with/or ol.geom.Polygon.
+   *
+   * @param {ol.feature | ol.geom.Polygon} polygon The polygon geometry to split.
+   * @param {ol.feature | ol.geom.LineString} lineFeat The line geometry to split the polygon
+   *  geometry with.
```

Comment:
Will thus also work for multi geometries?

Reply:
Does this work with (Multi-)polygons with holes?

Reply:
Yes it does. I'll add a test.